### ISSUE TITLE
Allow sources with 0 rows?

### ISSUE THREAD

**Issue by smilingthax** [evidence-dev/evidence#1850]
## Feature Description

Currently, `evidence sources` "skips" source tables with 0 rows:
```
  test1 ⚠ Finished. 0 rows, did not create table
```

This causes existing Pages / Queries which try to `SELECT ... FROM test1` to break:
```
Catalog Error: Table with name test1 does not exist!
```

The break is more "fundamental" than just `Dataset is empty - query ran successfully, but no data was returned from the database` (which can easily be allowed in charts by setting `emptySet=pass`).

## Goal of Feature

When copying an evidence project to use it against different source data, those databases are often initially almost empty.  
Saying **"No data"** would be *perfectly fine*, resulting in a hard red **"Error"** is *not.*

## Current Solution / Workarounds

Comment out affected parts in the pages.

## Alternatives

Maybe something like `{#if ...table...X...present} ... sql ... graph {/if}` ?? 

## Implementation

1. `buildMultipartParquet` in https://github.com/evidence-dev/evidence/blob/next/packages/lib/universal-sql/src/build-parquet.js#L71 should not return early:
```
	[...]
		if (currentBatch.length) await flush(currentBatch);   // -> must be called once unconditionally
// possible, hacky implementation: 
//		if (currentBatch.length || !tmpFilenames.length) await flush(currentBatch); 
	[...]
		if (currentBatch.length) await flush(currentBatch);   // -> same

	[...]
	if (!tmpFilenames.length) return 0;
```

2. Do not return before writing `.schema.json` file here: https://github.com/evidence-dev/evidence/blob/next/packages/lib/plugin-connector/src/data-sources/build-sources.js#L413  
   Basically the whole `if (!writtenRows) {` section would no longer be needed, because `writtenRows === 0` now also generates valid tables.

**Comment by hughess** [comment]
Closing this issue along with our release of the new Evidence Core: https://docs.evidence.dev/

Feel free to open a new issue for this if it is blocking you!
